Source/UltraleapTrackingCore/Private/LeapWrapper.cpp

```diff
@@ -508,13 +508,23 @@ void FLeapWrapper::AddDevice(const uint32_t DeviceID, const LEAP_DEVICE_INFO& De
 	AsyncTask(ENamedThreads::GameThread,
 		[this,DeviceInfo, DeviceID, DeviceHandle]()
 		{
+			DataLock->Lock();
```

Comment:
Is this a bug fix for another issue?

Reply:
A very intermittent crash occurred, that should fix it